implement-shell-tools/cat/cat.py

```diff
@@ -0,0 +1,29 @@
+import argparse
+
+parser = argparse.ArgumentParser(
+                    prog='Cat Command',
+                    description='Display and concatenate files contents')
+
+parser.add_argument("-n", help="Number the output lines, starting at 1.", default = None, action="store_true")
+parser.add_argument("-b", help="Number the non-blank output lines, starting at 1.", default = None, action="store_true")
```

Comment:
What happens if neither `-n` nor `-b` are used?
For instance, what will your solution print if we invoke it like this: `python3 cat.py sample-files/1.txt`?